CLI/test/CLITest.cs

```diff
@@ -77,31 +78,52 @@ public void TestExtractCopies()
 
         [TestMethod]
         public void TestReturnCode() {
-            CLITestHelper.Test("return_code_0", ReturnCode.Success);
-            CLITestHelper.Test("return_code_1", ReturnCode.FatalError);
```

Comment:
Why delete return_code1 ?  We need a test where the path of the error file is not found.

Reply:
A wrong path to error file error is now handeled in TestArgumentsErrors().